pkg/utils/docker.go

```diff
@@ -189,9 +188,11 @@ func PullImage(image string, jsonOutput bool) {
 func TagImage(source, tag string) {
 	out, err := exec.Command("docker", "tag", source, tag).Output()
 	errors.CheckErr(err, 102, "Image Tagging Failed")
-
-	output := string(out[:])
-	fmt.Println(output)
+	if len(out) > 0 {
+		// if 'out' byte array is ever > 0 then print the contents as string
+		output := string(out[:])
+		logr.Infoln(output)
+	}
```

Comment:
Is this to avoid a blank line in the log output ?

Reply:
Yes. A blank line looked weird when it is just `INFO[0001]`. With this logic it will only print output if there is any